app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class PostController extends Controller
+{
+    /**
+     * Show all blog posts.
+     */
+    public function index(): View
+    {
+        // ✅ Fetch all active posts
+        $posts = Post::where('is_active', true)->orderBy('created_at', 'desc')->get();
```

Comment:
You are using a Post model that is not in the current changes here on GitHub. Do I miss some of your code here?